src/RefractAPI.cc

```diff
@@ -177,15 +213,18 @@ namespace drafter {
     }
 
     template<typename T>
-    refract::IElement* ExtractParameter(const snowcrash::Parameter& parameter)
+    refract::IElement* ExtractParameter(const SectionInfo<snowcrash::Parameter>& parameter)
     {
         refract::IElement* element = NULL;
 
-        if (parameter.values.empty()) {
-            element = refract::IElement::Create(LiteralTo<T>(parameter.exampleValue));
+        if (parameter.section.values.empty()) {
+            element = refract::IElement::Create(LiteralTo<T>(parameter.section.exampleValue));
+            AttachSourceMap(element, MAKE_SECTION_INFO(parameter, exampleValue));
```

Comment:
Yes, I will replaceit.
It happens because I start with doing it "by hand". Later I created function `LiteralToRefract` because repeating code and I forget to change it there.